pages/Advanced-Setup.md

```diff
@@ -70,14 +70,15 @@ Example: if a user has a global "fly.use" permission, and then has a negated "fl
 
 * **Temporary permissions will override non-temporary permissions.**
 
-Example: if a user has a false permission set for "test.node", and a temporary true permission set for "test.node", the temporary permission will override the permanent one, and the user will be granted the true node.
+Example: if a user has a false permission set for "test.node", and a temporary true permission set for "test.node", the temporary permission will override the permanent one, and the user will be granted the true node for the duration of it.
 
 * **Wildcard/regex permissions will be overridden by normal permissions**
 
 Example: if a user has a true permission set for "luckperms.\*", and a false permission set for "luckperms.something", the non-wildcard permission will override the wildcard, and "luckperms.something" will be set to false, despite the wildcard.
 
 * **Temporary permissions will override other temporary permissions with a longer expiry time**
 
+Example: if a user has a temporary true permission set for "fly.use" that expires in 1 day, and a temporary false permission set for "fly.use" that expires in 1 hour, the temporary permission expiring in 1 hour will override the one expiring in 1 day, and the user will be granted the negative node for the duration of 1 hour.
```

Comment:
> the user will be granted the negative node

As the user has both nodes for that hour, it may be more clear to say something like "the negative node will take precedence" or "the negative node will apply"